graphql/schema.graphql

```diff
@@ -1,3 +1,4 @@
+# THIS FILE IS BEING KEPT FOR BACKWARDS COMPATABILITY.
```

Comment:
I'm confused by what this means?

Reply:
I deleted this file, I thought it would break Spruce frontend but it seems fine on staging